### DISCUSSION TITLE
GitHub repo issues after Sync fork with upstream repo merging many changes

### DISCUSSION THREAD

**Discussion by gahoekstraibm** [community/community#39857] [Repositories] [answered]
### Select Topic Area

Bug

### Body

After I did a "Sync fork" on below branch, viewing the branch on GitHub.com returns a 500 page or sometimes, the tree opens but with an error "Failed to load latest commit information." 

https://github.com/gahoekstraibm/flyway/tree/db2z

Trying to push a new commit from a local repository results in below error. Does this mean I have to recreate the GitHub repository?

Enumerating objects: 23, done.
Counting objects: 100% (23/23), done.
Delta compression using up to 8 threads
Compressing objects: 100% (6/6), done.
Writing objects: 100% (12/12), 824 bytes | 274.00 KiB/s, done.
Total 12 (delta 4), reused 9 (delta 1), pack-reused 0
remote: Resolving deltas: 100% (4/4), completed with 4 local objects.        
remote: fatal: did not receive expected object a343d7defee1b4b4792c8ade75c8f09c293be0e7        
To https://github.com/gahoekstraibm/flyway
 ! [remote rejected]     db2z -> db2z (failed)
error: failed to push some refs to 'https://github.com/gahoekstraibm/flyway'

**Comment by airtower-luna** [comment, marked as answer]
That sounds like *something* in the repository data on the GitHub end is messed up. I'd [send a bug report to GitHub support](https://support.github.com/contact/bug-report) so they can take a look and hopefully fix the problem.

**Reply by gahoekstraibm** [reply]
Ah, thanks, didn't realize there is a separate process for reporting bugs; just filed one.
